backend/services/implementations/userService.ts

```diff
@@ -16,6 +17,50 @@ const getMongoUserByAuthId = async (authId: string): Promise<User> => {
   return user;
 };
 
+/**
+ * Utility method for converting yyyy/mm/dd date
+ *  string to 24-char hex timestamp string
+ *
+ * ObjectID docs:
+ *  https://mongodb.github.io/node-mongodb-native/api-bson-generated/objectid.html
+ * Reference:
+ *  https://kchodorow.com/2011/12/20/querying-for-timestamps-using-objectids/
+ */
+const getDateTime = (date: string): string =>
+  `${Math.floor(new Date(date).getTime() / 1000).toString(16)}0000000000000000`;
+
+interface LimitByDateQuery {
```

Comment:
This interface isn't really limited to Limiting by date, is it? It's just a match query on the id property, that could be used for any kind of query on id no?